### ISSUE TITLE
test with the MCD6.1alpha, compilation with python 2.7

### ISSUE THREAD

**Issue by antoine-bierjon** [aymeric-spiga/mcd-python#44]
compile_fmcd.sh returns a file fmcd.so that looks ok ;
However quicktest.py fails, returns "fmcd.error: failed in converting 14th argument `extvarkeys' of fmcd.call_mcd to C/Fortran array"
=> probably because the inputs/outpus of call_mcd have changed since version 5.3, and they need to be updated in mcd.py (see MCD6.1 user manual)

**Comment by aymeric-spiga** [comment]
absolutely, I already had to change this between version 4 and version 5, since extvarkey became an array
